core/payment-driver/zksync/src/driver.rs

```diff
@@ -149,7 +150,7 @@ impl PaymentDriver for ZksyncDriver {
         //     return Err(GenericError::new("Can not init, account not active"));
         // }
 
-        wallet::init_wallet(&msg).await?;
+        wallet::init_wallet(&msg).timeout(Some(180)).await.map_err(GenericError::new)??;
```

Comment:
Maybe we can just use `std::time::Duration::from_secs(180)` here and delete the copied code?

Reply:
`.timeout()` does not exist without the copied code

Reply:
moved it to a neater place :)